### PR TITLE
Styleguide analyzer

### PR DESCRIPTION
**Pull request by TheReincarnator**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -115,19 +115,92 @@ className?: string;
...
 You can also specify the @name annotation on enum members, and you can add it to the props interface to rename the entire pattern.
 
-### Source-code structure and architecture
+### Styleguide analyzers ###
 
-Alva is a React application using MobX as state management and Electron to provide a stand-alone application.
+Alva tries to understand the structure of your styleguide, including the pattern folders, patterns, and properties, by delegating to so-called styleguide analyzers.
 
-Additionally, it ships with a living styleguide project, which consists of Patternplate React components.
+In the future, there are several analyzers for all types of
```
Comment (LKuechler) [inline README.md]: `In the future, there will be...`

File: README.md [hunk 2]
```diff
@@ -115,19 +115,92 @@ className?: string;
...
+* frontend technology like [React](https://reactjs.org/), [Angular](https://angular.io/), and [Vue](https://vuejs.org/),
```
Comment (LKuechler) [inline README.md]: `... frontend technologies ...`

File: README.md (hunk 1 above)
Comment (TheReincarnator) [reply README.md]: done.

File: README.md (hunk 2 above)
Comment (TheReincarnator) [reply README.md]: done.

**Pull request merged**